native/packages/react-native-bpk-component-phone-input/src/BpkDiallingCodeList.ios.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { View, StyleSheet, SectionList } from 'react-native';
+import React from 'react';
+import BpkText from 'react-native-bpk-component-text';
+import {
+  colorGray50,
+  colorGray100,
+  spacingSm,
+  spacingMd,
+} from 'bpk-tokens/tokens/base.react.native';
+import { BpkDiallingCodeListItem } from '..';
+import {
+  type ListCommonProps,
+  LIST_COMMON_PROP_TYPES,
+  LIST_COMMON_DEFAULT_PROPS,
+} from './common-types';
+
+const spacing = spacingMd * 1.5;
```

Comment:
This should be spacingBase(16pt) and maybe it could be given a more descriptive name?

Reply:
I've changed `spacingMd * 1.5` to `spacingBase` across the board and it looks too spaced out now:

<img width="244" alt="screen shot 2018-03-02 at 08 48 56" src="https://user-images.githubusercontent.com/73652/36890637-90aa1b56-1df6-11e8-8857-9bec06bb0249.png">

I should also point out that when I spoke to James about this before he left, he said he'd just used values from iOS and not Backpack tokens, which is why we agreed to hack it together with `spacingMd * 1.5`. In Sketch, the vertical padding is 12, hence the calculation.

What do you reckon?

Reply:
Yes agree we should use the iOS defaults, but the iOS default is 16pt AFAIK. That's also what I am seeing in Sketch with the UI library builtin to 49

Reply:
Also just to be clear I'm only talking about the leading margin being 16pt here not the vertical padding of the cell. Cells should be 44pt high if we align with the default

Reply:
Ah yeah I see it a lot more now I'm testing on device. I've updated it accordingly!